single.php

```diff
@@ -111,6 +111,13 @@
 		   </div>
 		  <?php 	}	?>
 
+	<?php
+	if ( get_field( 'calendar_url' ) ) {
+		$calendar_url = get_field( 'calendar_url' );
+		echo '<h2>For more about this event, please visit <a href="' . esc_url( $calendar_url ) . '">' . esc_url( $calendar_url ) . '</a></h2>';
```

Comment:
Any reason this is an h2? Might look nicer as a p... (Not that anyone should end up looking at it anymore.)

Reply:
Yeah, I went back and forth on this. I went with an h2 because I'd rather the user get something biggish to help them along, if they do in fact make it to this page. Maybe there's another choice, though.

As an h2
![image](https://user-images.githubusercontent.com/1403248/36449569-265361c6-1659-11e8-8fb8-d184da425f3f.png)

As a p
![image](https://user-images.githubusercontent.com/1403248/36449592-3d3a91f2-1659-11e8-84da-c1461c7f4c30.png)

Reply:
I don't think it's worth the extra code to add spacing to the h2, but let's just go with that.

Reply:
Sounds good to me - we can reconsider all of this if we get around to rebuilding the theme, but I'm not crossing my fingers at this point.